### PR TITLE
[`refurb`] Implement `unnecessary-from-float` (`FURB164`)

### PR DESCRIPTION
**Pull request by boolean-light**
## Summary

Implement FURB164 in the issue #1348.
Relevant Refurb docs is here: https://github.com/dosisod/refurb/blob/v2.0.0/docs/checks.md#furb164-no-from-float

I've changed the name from `no-from-float` to `verbose-decimal-fraction-construction`.

## Test Plan

I've written it in the `FURB164.py`.

### CODE REVIEW

**Comment by charliermarsh** [comment]
Maybe `verbose_fraction_construction`?

**Comment by boolean-light** [comment]
I have thought of that, but should we incorporate `Decimal` as it is checks for both constructions of `Decimal` and `Fraction`?

**Comment by boolean-light** [comment]
Also I'm not very sure why `cargo test` for wasm is failing... Let me check on that.

**Comment by MichaReiser** [comment]
It sometimes has this error. Let me re-run it to see if it resolves the issue.

File: crates/ruff_linter/src/rules/refurb/rules/verbose_decimal_fraction_construction.rs [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+/// Fraction.from_float(4.2)
+/// Fraction.from_decimal(Decimal("4.2"))
+/// ```
+///
+/// Use instead:
+/// ```python
+/// Decimal(4.2)
+/// Decimal("inf")
+/// Fraction(4.2)
+/// Fraction(Decimal("4.2"))
```
Comment (charliermarsh) [inline crates/ruff_linter/src/rules/refurb/rules/verbose_decimal_fraction_construction.rs:36]: Should the inner `"4.2"` be `4.2`?

**Review by charliermarsh — APPROVED** [review]
Thanks!

**Comment by boolean-light** [comment]
Sorry for the late response and thanks for the review!

**Pull request merged**